### PR TITLE
Cleanup and add API docs.

### PR DESCRIPTION
**Pull request by kasiastreich**

### CODE REVIEW

File: workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/ConfidentialMoveFungibleTokensFlow.kt [hunk 1]
```diff
@@ -11,7 +11,19 @@ import net.corda.core.identity.AbstractParty
...
-class ConfidentialSelectAndMoveFungibleTokensFlow<T : TokenType>
+/**
+ * Version of [MoveFungibleTokensFlow] using confidential identities. Confidential identities are generated and exchanged for
+ * all parties that receive tokens states.
+ * Call this for one [TokenType] at a time. If you need to do multiple token types in one transaction then create a new
+ * flow, calling [addMoveTokens] for each token type and handle confidential identities exchange yourself.
+ *
+ * @param partiesAndAmounts list of pairing party - amount of token that is to be moved to that party
+ * @param participantSessions sessions with the participants of move transaction
+ * @param observerSessions optional sessions with the observer nodes, to witch the transaction will be broadcasted
```
Comment (roger3cev) [inline workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/ConfidentialMoveFungibleTokensFlow.kt]: ... to which the ...

File: workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/ConfidentialMoveNonFungibleTokensFlow.kt [hunk 2]
```diff
@@ -10,7 +10,18 @@ import net.corda.core.flows.FlowSession
...
-class ConfidentialSelectAndMoveNonFungibleTokensFlow<T : TokenType>
+/**
+ * Version of [MoveNonFungibleTokensFlow] using confidential identities. Confidential identities are generated and exchanged for
+ * all parties that receive tokens states.
+ * Call this for one [TokenType] at a time. If you need to do multiple token types in one transaction then create a new
+ * flow, calling [addMoveTokens] for each token type and handle confidential identities exchange yourself.
+ *
+ * @param partyAndToken list of pairing party - token that is to be moved to that party
+ * @param participantSessions sessions with the participants of move transaction
+ * @param observerSessions optional sessions with the observer nodes, to witch the transaction will be broadcasted
```
Comment (roger3cev) [inline workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/ConfidentialMoveNonFungibleTokensFlow.kt]: which

File: workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/MoveFungibleTokensFlow.kt [hunk 3]
```diff
@@ -9,10 +9,18 @@ import net.corda.core.node.services.vault.QueryCriteria
...
+ * Inlined flow used to move amounts of tokens to parties, [partiesAndAmounts] specifies what amount of tokens is moved
+ * to each participant with possible change output paid to the [changeOwner].
  * Call this for one [TokenType] at a time. If you need to do multiple token types in one transaction then create a new
  * flow, calling [addMoveTokens] for each token type.
- */
-class SelectAndMoveFungibleTokensFlow<T : TokenType>
+ *
+ * @param partiesAndAmounts list of pairing party - amount of token that is to be moved to that party
+ * @param participantSessions sessions with the participants of move transaction
+ * @param observerSessions optional sessions with the observer nodes, to witch the transaction will be broadcasted
```
Comment (roger3cev) [inline workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/MoveFungibleTokensFlow.kt]: which

File: workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/ConfidentialMoveFungibleTokensFlow.kt (hunk 1 above)
Comment (kasiastreich) [reply workflow/src/main/kotlin/com/r3/corda/sdk/token/workflow/flows/move/ConfidentialMoveFungibleTokensFlow.kt]: Freudian slip?

**Pull request merged**